src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java

```diff
@@ -71,4 +75,8 @@ public MixinEntityPlayer(World worldIn) {
         return this.openContainer == this.inventoryContainer;
     }
 
+    public String human$getName() {
+        return this.getName();
```

Comment:
A new way of fixing this error has just been pushed to the Mixin repo, you can use an [`@Intrinsic`](https://github.com/SpongePowered/Mixin/blob/master/src/main/java/org/spongepowered/asm/mixin/Intrinsic.java) annotation.